завершил первую страницу

vishnyakovvv opened the pull request.

vishnyakovvv committed:
Добавил название проекта, данные студента и наставника

vishnyakovvv committed:
добавил header

немного ввел в ступор "поиск"

vishnyakovvv committed:
Update index.html

заполняю первую страницу сайта

vishnyakovvv committed:
первая страница

раздел Delivery не готов

vishnyakovvv committed:
завершил первую страницу

не понял что прописывать в заголовке service

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,260 @@
+      <section class="slideshow-container">
+        <div class="mySlides-01">
+          <div class="mySlides-01-number">01</div>
+          <h1 class="mySlides-01-title">THE OOH NOO COLLECTION</h1>
```
uppercase-ом писать не нужно, это все делается потом стилями

vishnyakovvv replied:
а uppercase-ом это значит заглавными?

ea-dorosh replied:
да

vishnyakovvv replied:
дело в том что я копировал текст с проекта напрямую, а там как раз заглавные))

ea-dorosh replied:
Значит нужно руками текст переписать, прописных букв не должно быть в проекте

vishnyakovvv replied:
исправил

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,260 @@
+          <div class="mySlides-01-number">01</div>
+          <h1 class="mySlides-01-title">THE OOH NOO COLLECTION</h1>
+          <p class="mySlides-01-text">Designed in Slovenia</p>
+          <img src="img1.jpg" alt="COLLECTION" />
```
/ такая палочка не нужна в конце. img это одиночный тэг и закрывать его никак не нужно

vishnyakovvv replied:
этот слеш всегда появляется после сохранения ctrl+s, я тоже обращал на это внимание, но подумал, может там плагин правильно выравнивает. можно убрать эту функцию?

ea-dorosh replied:
можешь поискать как убрать ее в vs коде, по идее это что-то про auto collapse
но на верстку не влияет, так что это не очень важно, но я бы убрал

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,260 @@
+          <button class="mySlides-01-button">EXPLORE NOW</button>
+          <div class="mySlides-01-bottomtext">
+            Free (continental) USA shipping on all orders of $100 USD or more!
+          </div>
```
это по идее не див, а текстовый фрагмент, лучше 'p'

vishnyakovvv replied:
исправил

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,260 @@
+      <section class="slideshow-container">
+        <div class="mySlides-01">
+          <div class="mySlides-01-number">01</div>
+          <h1 class="mySlides-01-title">THE OOH NOO COLLECTION</h1>
```
здесь не h1, а h2
так как это слайды их несколько и значит это противоречит принципу одного h1 на странице

vishnyakovvv replied:
исправил

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,260 @@
+          <li class="category-card">
+            <img src="APPAREL_KIDS.png" alt="APPAREL KIDS" />
+            <a class="category-card-product" href="#">APPAREL KIDS</a>
+          </li>
```
здесь скорее ссылка оборачивает в себе картинку а не соседствует с ней, я бы сразу обернул, но можешь и позже сделать, пока не так важно

vishnyakovvv replied:
исправил

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,260 @@
+        </div>
+      </section>
+      <section class="category">
+        <h1 class="category-title">CATEGORY PRODUCTS</h1>
```
h2

vishnyakovvv replied:
исправил

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,260 @@
+        </p>
+      </section>
+      <section class="service">
+        <h1 class="servic-tittle">???</h1>
```
h2

vishnyakovvv replied:
исправил

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,260 @@
+              </p>
+            </div>
+          </li>
+        </ul>
```
можно пока так оставить, но я думаю здесь список из меню соседствует со списком с  описанием и видимо рядом еще список с картинками. Потому что как ты сейчас сделал так не стилизуешь как в макете

vishnyakovvv replied:
к этому блоку у меня больше всего вопросов, так что давай пока оставим так, но чуть позже вникну и исправлю

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,260 @@
+        </ul>
+      </section>
+      <section class="design">
+        <h1 class="design-title">DESIGN LIFE KIDS</h1>
```
h2

vishnyakovvv replied:
исправил

ea-dorosh commented on index.html:
```diff
@@ -0,0 +1,260 @@
+        <button class="design-button">BUY NOW</button>
+      </section>
+      <section class="contacts">
+        <h1 class="contacts-tittle">CONTACTS</h1>
```
h2

vishnyakovvv replied:
исправил

vishnyakovvv committed:
исправил ошибки

vishnyakovvv committed:
исправил синтаксис

не могу найти как убрать эту функцию, добавляются не только лишние пробелы и слеш, но и идет перенос тегов и т.д.

vishnyakovvv committed:
добавлена вторая страница

не смог я найти как отключть доп пробелы и доп слэш, так там еще img вообще каряво сохраняет, обрати внимание.  завтра с буком на учебу пойду, может Женя поможет разобратьтся

vishnyakovvv committed:
внес поправки

vishnyakovvv committed:
заполнил страницу HTML
